openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -523,13 +570,17 @@ OpenAssessment.ResponseView.prototype = {
     },
 
     /**
-     Set the image URL, or retrieve it.
+     Set the file URL, or retrieve it.
      **/
-    imageUrl: function() {
+    fileUrl: function() {
         var view = this;
-        var image = $('#submission__answer__image', view.element);
+        var file = $('#submission__answer__file', view.element);
         view.server.getDownloadUrl().done(function(url) {
-            image.attr('src', url);
+            if (file.prop("tagName") == "IMG") {
```

Comment:
I don't think there is any issue when comparing string. Though I'll change it to be more clear.